decisions/0006-program-slugs-in-publisher.rst

```diff
@@ -0,0 +1,32 @@
+6. Program Slugs in Publisher
+-------------------------------
+
+Status
+------
+
+Accepted (July 2023)
+
+Context
+-------
+
+Ability to update the slugs for programs is needed to be accessible to a group of users.
+These slugs are used in generating url for each type of programs.
+These program slugs will be programmatically generated, just like courses.
+In order to grant the ability to update slugs for programs, we're going to display programs on Publisher due to it's ease of access control.
+Instead of displaying all program information on the publisher, we will only show the editable slug field.
+
+Decision
+--------
+
+* A new tab will be added on Publisher named "Programs/Degrees" in the header
+* Upon clicking the tab, the user will be shown a list of programs with minimal info
+* Each item in the list will take the user to it's detail page where an input field will be available for slugs and a save button
+* To access the program list and detail pages, every user must be granted the necessary permissions
+
+
+Consequences
+------------
+
+* Programs will be available on Publisher
+* Authorized users will have the ability to edit program slugs, facilitating the creation of marketing URLs with proper formatting
+* Proper access control will ensure that only authorized users can view and modify programs
```

Comment:
These two points are not consequences. They should be moved to "Decision" and should focus on how these will be carried out.

Reply:
Should I remove these altogether as these are already mentioned in decisions?